test/container_registry.go

```diff
@@ -0,0 +1,67 @@
+package test
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/release-utils/command"
+)
+
+const (
+	testDockerfile = `FROM scratch
+CMD [""]
+`
+	testDockerfileName    = "Dockerfile"
+	registryImage         = "registry:2"
+	registryContainerName = "k8s-registry"
+)
+
+type dockerRegistry struct {
+	ImageName      string
+	DockerfilePath string
+}
+
+func RunDockerRegistryWithDummyImage(t *testing.T, imageName string) *dockerRegistry {
+	// Setup the Docker Registry
+	cmd := command.New("docker", "run", "--detach", "--network", "host",
```

Comment:
Const the docker command here or golangci is going the get angry at you:
```suggestion
	cmd := command.New(dockerCommand, "run", "--detach", "--network", "host",
```

Reply:
Good point. Changed, hope golangci is not going to be angry at me. :crossed_fingers: